src/Sylius/Behat/Page/Shop/Checkout/PaymentPageInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shop\Checkout;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+interface PaymentPageInterface
+{
+    public function nextStep();
```

Comment:
I would place this after ``selectPaymentMethod``, as we firstly select payment method and continue to next step after that 😄